fmfexporter/connectors/jira/fmf_jira.py

```diff
@@ -0,0 +1,108 @@
+import configparser
```

Comment:
Please move this connectors package inside the Polarion adapter.

Reply:
I don't think this is connected to polarion in any way imo.
You can have defects in JIRA, Bugzilla, Redmine etc. Not related to polarion. And those are defects tracking systems. So I would rather keep it as is. (more discussion? :) )

Reply:
It's not defined genetically. The goal of this till is exporting test cases, so if we need connector hooks, I suggest defining a generic connector hook interface and new methods on the generic adapter, so it's not tightly coupled with a specific implementation.

Reply:
Moved to polarion module, to stay KISS. If we would need more conenctors, I would create abstraction over it.